prep-example/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>prep-example</title>
+</head>
+<body>
+  <h1>I would like to build</h1>
+  <p>
+    <!-- write your app ideas here 👇 -->
+
+
+
```

Comment:
Hey Nick! When you have a sec, it looks like your text was not included for the commit

Reply:
Of corse I deleted the wrong one

Reply:
I imported 2 of the same example files into vsc